aws/src/integration/java/org/apache/iceberg/aws/s3/TestS3FileIOIntegration.java

```diff
@@ -109,17 +113,23 @@ public static void beforeClass() {
     AwsIntegTestUtil.createAccessPoint(s3Control, accessPointName, bucketName);
     AwsIntegTestUtil.createAccessPoint(
         crossRegionS3Control, crossRegionAccessPointName, crossRegionBucketName);
-    s3.putBucketVersioning(
-        PutBucketVersioningRequest.builder()
-            .bucket(bucketName)
-            .versioningConfiguration(
-                VersioningConfiguration.builder().status(BucketVersioningStatus.ENABLED).build())
-            .build());
+    if (!checkIfS3Express()) {
+      s3.putBucketVersioning(
+          PutBucketVersioningRequest.builder()
+              .bucket(bucketName)
+              .versioningConfiguration(
+                      VersioningConfiguration.builder().status(BucketVersioningStatus.ENABLED).build())
+              .build());
+    }
   }
 
   @AfterAll
   public static void afterClass() {
-    AwsIntegTestUtil.cleanS3Bucket(s3, bucketName, prefix);
+    if (checkIfS3Express()){
```

Comment:
Is this section worth moving to the utils class?

Reply:
hmm I could keep the cleanS3Bucket name and then just split it to s3 general or s3 express based on this check in there? do you prefer that?

Reply:
Yeah. It's up to you though, I don't have a strong opinion on this